1_fetch.R

```diff
@@ -86,6 +87,25 @@ p1_targets_list <- list(
   tar_target(
     p1_reaches_sf,
     st_read(p1_reaches_shp)
+  ),
+  
+  # Download NHDPlusV2 flowlines for DRB
+  tar_target(
+    p1_nhdv2reaches_sf,
```

Comment:
This is not critical, but one comment about regarding the set-up of targets. From my (new) reviewer perspective, it might be easier to put this piped process into another stand alone function - maybe within `get_nhdv2_flowlines.R`. There are quite a few steps in the dplyr pipe that could be compartmentalized, and also, the function `getnhdv2_flowlines` is a bit hard to catch and review here, since it's within in the pipe (same can be said about the `p2_prms_nhdv2_xwalk` target,  where two functions are called from `pair_nhd_reaches.R`. Just an idea here.

Reply:
> From my (new) reviewer perspective, it might be easier to put this piped process into another stand alone function - maybe within get_nhdv2_flowlines.R

Thanks, @msleckman. I agree with your suggestion to move the code currently in the phase file into a separate function. Now all of the manipulation steps occur within `2_process/get_nhdplusv2.R` and the only line within the target is the call to `get_nhdv2_flowlines` Changes are in this commit: https://github.com/USGS-R/drb-inland-salinity-ml/pull/35/commits/629f235be5f8dc980893ba06a68f664f5b806013